packages/components-library/src/components/filters/ActiveFilters.vue

```diff
@@ -0,0 +1,190 @@
+          // Checkbox
+          if (filter.type === 'checkbox-filter') {
+            // resolve options function/promise and show results later
+            const option = await filter.options()
+            current.forEach(subKey => {
+              const findTextFromValue = option.filter(
+                filterOption => filterOption.value === subKey
+              )[0]
+              activeValues.push({
+                key,
+                subKey,
+                value: findTextFromValue.text,
+                label: filter.label
+              })
+            })
+          }
+          // Range Filter
+          if (filter.type === 'range-filter') {
+            if ((current[0] === null || current[0] === '') && current[1] != null) {
+              activeValues.push({
+                key,
+                value: `${current[1]} and less`,
+                label: filter.label
+              })
+            } else if (current[0] !== null && (current[1] === null || current[1] === '')) {
+              activeValues.push({
+                key,
+                value: `${current[0]} and more`,
+                label: filter.label
+              })
+            } else if (current[0] !== null && current[1] !== null) {
+              activeValues.push({
+                key,
+                value: `${current[0]} to ${current[1]}`,
+                label: filter.label
+              })
+            }
+          }
+          if (filter.type === 'multi-filter') {
+            const options = await filter.options({ nameAttribute: false, queryType: 'in', query: current.join(',') })
+            current.forEach(subKey => {
+              const findTextFromValue = options.filter(
+                filterOption => filterOption.value === subKey
+              )[0]
+              activeValues.push({
+                key,
+                subKey,
+                value: findTextFromValue.text,
+                label: filter.label
+              })
+            })
+          }
+        } else {
+          // Single item
+          activeValues.push({ key, value: current, label: filter.label })
+        }
+      })
+      if (this.value === newValue) {
+        this.activeValues = activeValues
+      }
+    },
+    selectFilter (key) {
+      return this.filters.filter(filter => filter.name === key)[0]
+    },
+    removeFilter ({ key, subKey }) {
+      const selections = { ...this.value }
+      if (subKey === undefined) {
+        delete selections[key]
+      } else {
+        selections[key] = selections[key].filter(selectionKey => selectionKey !== subKey)
+      }
+      this.$emit('input', selections)
+    }
+  }
+})
+</script>
+
+<style scoped>
+button svg path {
+  transition: fill 0.3s;
+}
+button:hover svg path {
+  fill: var(--danger);
+}
+</style>
+
+<docs>
+Visualisation of active filters
+
+### Usage
+```jsx
+
+const selections = {
+  search: 'test',
+  name: 'John',
+  age: [32, 45],
+  fruit: 'orange',
+  color: ['red', 'green']
+}
```

Comment:
Are you sure this covers all use-cases of the component? It seems there is also something related to datetime?

Reply:
added